chat_exporter/construct/assets/component.py

```diff
@@ -44,15 +44,15 @@ async def build_component(self, c):
             Component.menu_div_id += 1
 
     async def build_button(self, c):
-        url = c.url if c.url else ""
+        url = f"<a href='{c.url}' style='text-decoration:none' target='_blank'>" if c.url else "<a href='javascript:;' style='text-decoration:none'>"
```

Comment:
what is the idea behind this change?

Reply:
Before, clicking a non-url button would refresh the page which I and many others found annoying. This change makes it so that the button is still "hoverable" but won't refresh the page when clicked, and still allows url-buttons to function and instead opens the link in a new tab.